README.md

```diff
@@ -23,33 +23,36 @@ compiler options:
 -fplugin=Smuggler.Plugin
 ```
 
-You can also use `smuggler` with `ghcid` to enable on-the-fly features:
+Lovely addition to this package is that it automatically supports on-the-fly
+feature if you use it with `ghcid`. Smuggler doesn't perform file changes when
+there are no unused imports. So you can just run `ghcid` as usual:
 
 ```
-ghcid --command='cabal new-repl --ghc-options="-fplugin=Smuggler.Plugin"'
+ghcid --command='cabal repl'
 ```
 
 ## For contributors
 
 Requirements:
 
-* `ghc-8.6.3`
+* `ghc-8.6.5`
+* `cabal >= 3.0` or `stack >= 2.0`
```

Comment:
Isn't it too tight? I guess it could work with 2.4 as well (but this requires to specify in docs `new` and ordinary commands, which is overhead...)

Reply:
Yeah, I specified `>= 3.0` just not to add `new-` or `v2-prefix`..